.metalinter.json

```diff
@@ -1,7 +1,8 @@
 {
   "Linters": {
     "unused":   "unused -tags integration:PATH:LINE:COL:MESSAGE",
-    "gosimple": "gosimple -tags integration:PATH:LINE:COL:MESSAGE" },
+    "gosimple": "gosimple -tags integration:PATH:LINE:COL:MESSAGE",
```

Comment:
Could you clean this up while you're here, gometalinter supports the following syntax iirc:

```
"unused": { "Command": "unused -tags integration" },
...
```